packages/shared-components/src/hooks/useStudentResponse.ts

```
@@ -49,6 +50,21 @@ function useStudentResponse({
               valid: true,
             },
           }
+        } else if (element.elementData.type === ElementType.Selection) {
+          const emptyResponses = getEmptySelectionResponse({
+            numberOfInputs: (element.elementData as SelectionElementData)
+              .options.numberOfInputs,
+          })
+
+          return {
+            ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)